components/celplace/src/main/java/com/celements/crm/place/geocoding/GeocodingService.java

```diff
@@ -56,9 +59,22 @@ GeoApiContext getGMapsContext() {
     return gMapsContext;
   }
 
+  @Override
+  public ClassDefinition getGeotagClass() {
+    String geotagClassName = configSource.getProperty(
+        "celements.geocoding.celementsPlaces.geotagClassName", String.class);
+    ClassDefinition classDef;
+    if (!Strings.isNullOrEmpty(geotagClassName)) {
+      classDef = Utils.getComponent(ClassDefinition.class, geotagClassName);
```

Comment:
there might not necessarily exist a component for the configured `geotagClassName`. in those cases this code produces a `RuntimeException`, see `Utils#getComponent`.
instead do the following:
1. check `#hasComponent` for the given `geotagClassName`
1. if false set default `geotagClassName` instead
1. lookup component with the now valid `geotagClassName`